test/plugins/builds.test.js

```diff
@@ -1339,6 +1341,8 @@ describe('build plugin test', () => {
                             scmContext,
                             eventId: 'bbf22a3808c19dc50777258a253805b14fb3ad8b',
                             configPipelineSha,
+                            prSource: '',
+                            prInfo: '',
```

Comment:
This is not correct test case. This test case is the chainPRs test case. Therefore, in this test case, `prSource` and `prInfo` should have a value.

Reply:
@kumada626 @ibu1224 can this be fixed in a subsequent PR. I need to merge this since we need to get an internal release out for unblocking Parallel join.